--- index.php	2004-07-12 16:08:41.350777600 -0400
+++ index-admin_mod.php	2004-07-12 17:25:21.455444800 -0400
@@ -357,17 +357,22 @@
 
 function show_login_box()
 {
-	global $this_file, $subdir;
-	$sd = translate_uri($subdir);
-	echo '<p /><table border="0" cellpadding="8" cellspacing="0">
-	<tr class="paragraph"><td class="default_td">Login:',
-	"\n<form method='post' action='{$this_file}dir=$sd'>
-	<table><tr class=\"paragraph\"><td>Username:</td>
-	<td><input type='text' name='user' />
-	</td></tr><tr class=\"paragraph\"><td>Password:</td>
-	<td><input type='password' name='pass' /></td></tr></table>
-	<p><input class='button' type='submit' value='Login' /></p>
-	</form></td></tr></table>";
+	if (isset($_GET['show_login']))
+	//you can change 'show_login' to whatever you want to add to the URL
+	//to access the login box, go to index.php?show_login
+	{
+		global $this_file, $subdir;
+		$sd = translate_uri($subdir);
+		echo '<p /><table border="0" cellpadding="8" cellspacing="0">
+		<tr class="paragraph"><td class="default_td">Login:',
+		"\n<form method='post' action='{$this_file}dir=$sd'>
+		<table><tr class=\"paragraph\"><td>Username:</td>
+		<td><input type='text' name='user' />
+		</td></tr><tr class=\"paragraph\"><td>Password:</td>
+		<td><input type='password' name='pass' /></td></tr></table>
+		<p><input class='button' type='submit' value='Login' /></p>
+		</form></td></tr></table>";
+	}
 }
 
 function show_search_box()
