--- index.php	2004-06-01 14:41:11.497916800 -0400
+++ index-admin_mod.php	2004-06-05 13:33:55.995904000 -0400
@@ -342,15 +342,20 @@
 
 function show_login_box()
 {
-	global $this_file, $subdir;
-	$sd = translate_uri($subdir);
-	echo '<p><table border="0" cellpadding="8" cellspacing="0">
-	<tr class="paragraph"><td class="default_td">Login:<br />'
-	."\n<form method='post' action='{$this_file}dir=$sd'><p>
-	<table><tr><td>Username:</td><td><input type='text' name='user' />
-	</td></tr><tr><td>Password:</td><td><input type='password' name='pass' /></td></tr>
-	</table></p><p><input class='button' type='submit' value='Login' /></p>
-	</form></td></tr></table></p>";
+	if (isset($_GET['show_login']))
+	//you can change 'show_login' to whatever you want to add to the URL
+	//to access the login box, go to index.php?show_login
+	{
+		global $this_file, $subdir;
+		$sd = translate_uri($subdir);
+		echo '<p><table border="0" cellpadding="8" cellspacing="0">
+		<tr class="paragraph"><td class="default_td">Login:<br />'
+		."\n<form method='post' action='{$this_file}dir=$sd'><p>
+		<table><tr><td>Username:</td><td><input type='text' name='user' />
+		</td></tr><tr><td>Password:</td><td><input type='password' name='pass' /></td></tr>
+		</table></p><p><input class='button' type='submit' value='Login' /></p>
+		</form></td></tr></table></p>";
+	}
 }
 
 function show_search_box()
